app/assets/javascripts/projects/index.js

```diff
@@ -7,7 +7,7 @@
 // - refresh project users tab after manage user modal is closed
 // - refactor view handling using library, ex. backbone.js
 
-/* global Comments CounterBadge animateSpinner */
+/* global Comments CounterBadge animateSpinner initFormSubmitLinks HelperModule */
```

Comment:
Try adding `$` to this list, if it'll remove the `'$' is not defined no-undef` warnings.

Reply:
Fixed Hound config